### ISSUE TITLE
[BUG] z/OS datasets not deleted when deleting repository

### ISSUE THREAD

**Issue by andrew890** [wizardofzos/zigi#393]
**Describe the bug**
When deleting a repository you get the prompt:

Keep the z/OS Datasets     Y to keep, or N to delete

I enter "N" but the z/OS datasets are not deleted.

**Release of zigi**
3.10

**To Reproduce**
1.  Clone a repository with z/OS datasets
2.  Delete it using Zigi

**Expected behavior**
z/OS datasets are deleted

**Comment by lbdyck** [comment]
This should be resolved in 3.10

**Comment by andrew890** [comment]
Is that in the wizardofzos/zigi 3.10 branch? I just cloned that and installed it and observed the problem.

**Comment by lbdyck** [comment]
Yes - that's the 3.10 branch that I meant. I will try again tomorrow to recreate.

**Comment by lbdyck** [comment]
I've recreated this issue using your test repo but since your repo wasn't in zigi format that may have been the reason.  i will investigate over the weekend.

**Comment by lbdyck** [comment]
This should be fixed in the 3.10 branch

**Comment by andrew890** [comment]
The zigi branch is in zigi format

**Comment by lbdyck** [comment]
Using the test branch you provided I've not been able to recreate the issue.
